rust/worker/Cargo.toml

```diff
@@ -51,6 +51,7 @@ tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 opentelemetry = { version = "0.19.0", default-features = false, features = ["trace", "rt-tokio"] }
 opentelemetry-otlp = "0.12.0"
 shuttle = "0.7.1"
+tracing-panic = "0.1.2"
```

Comment:
Would rather we just DIY - this lib is small and not widely adopted and this seems like an contact point that would be useful to own the src of 

https://github.com/LukeMathWalker/tracing-panic/blob/e27729b9d7fe38d5c67db75f096f79328c8977c9/src/lib.rs#L60